app/models/rating.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (sequelize, DataTypes) => {
+  const Rating = sequelize.define(
+    'Rating',
+    {
+      ratingUserId: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+        onUpdate: 'cascade',
+        onDelete: 'set null'
```

Comment:
https://sequelize.org/master/manual/assocs.html#one-to-many-relationships
This is the default behavior